--- install-build-deps.sh
+++ install-build-deps.sh.new
@@ -102,7 +102,7 @@ fi
 
 distro_codename=$(lsb_release --codename --short)
 distro_id=$(lsb_release --id --short)
-supported_codenames="(trusty|xenial|bionic|disco|eoan)"
+supported_codenames="(trusty|xenial|bionic|disco|eoan|focal)"
 supported_ids="(Debian)"
 if [ 0 -eq "${do_unsupported-0}" ] && [ 0 -eq "${do_quick_check-0}" ] ; then
   if [[ ! $distro_codename =~ $supported_codenames &&
@@ -200,11 +200,9 @@ dev_list="\
   perl
   pkg-config
   python
-  python-cherrypy3
   python-crypto
   python-dev
   python-numpy
-  python-opencv
   python-openssl
   python-psutil
   python-yaml
@@ -242,7 +240,7 @@ common_lib_list="\
   libdrm2
   libevdev2
   libexpat1
-  libffi6
+  libffi7
   libfontconfig1
   libfreetype6
   libgbm1
@@ -481,25 +479,9 @@ elif package_exists libbrlapi0.6; then
 else
   dev_list="${dev_list} libbrlapi0.5"
 fi
-if package_exists apache2.2-bin; then
-  dev_list="${dev_list} apache2.2-bin"
-else
-  dev_list="${dev_list} apache2-bin"
-fi
 if package_exists libav-tools; then
   dev_list="${dev_list} libav-tools"
 fi
-if package_exists php7.3-cgi; then
-  dev_list="${dev_list} php7.3-cgi libapache2-mod-php7.3"
-elif package_exists php7.2-cgi; then
-  dev_list="${dev_list} php7.2-cgi libapache2-mod-php7.2"
-elif package_exists php7.1-cgi; then
-  dev_list="${dev_list} php7.1-cgi libapache2-mod-php7.1"
-elif package_exists php7.0-cgi; then
-  dev_list="${dev_list} php7.0-cgi libapache2-mod-php7.0"
-else
-  dev_list="${dev_list} php5-cgi libapache2-mod-php5"
-fi
 
 # Some packages are only needed if the distribution actually supports
 # installing them.
